doc/functions/html_cva.rst

```diff
@@ -48,10 +48,10 @@ Then use the ``color`` and ``size`` variants to select the needed classes:
     // class="alert bg-red text-md"
 
     {{ include('alert.html.twig', {'color': 'green', 'size': 'sm'}) }}
-    // class="alert bg-green text-sm"
+    // class"alert bg-green text-sm"
```

Comment:
Not sure if this should also be changed. But if so, it should probably be like this:
```suggestion
    // class: "alert bg-green text-sm"
```

Reply:
It's not good either, since we display a prevew of the rendered HTML. It must be reverted